stock_picking_move_done_manual/models/stock_move.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2020 ForgeFlow, S.L. (https://www.forgeflow.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, models
+
+
+class StockMove(models.Model):
+    _inherit = "stock.move"
+
+    def _action_done_get_picking(self, moves, moves_todo):
+        picking = super(StockMove, self)._action_done_get_picking(
+            moves, moves_todo)
+        for move in moves.filtered(lambda m: m.state != 'cancel'):
```

Comment:
Maybe you could ensure all moves have same picking, like moves.filtered(...).mapped('picking_id'). 
Code works anyway.